game-core/src/test/java/games/strategy/triplea/delegate/ITestDelegateBridge.java

```diff
@@ -11,8 +10,6 @@
 interface ITestDelegateBridge extends IDelegateBridge {
   void setStepName(String name);
 
-  void setRandomSource(IRandomSource randomSource);
-
```

Comment:
Any Idea on how complex it would be to get id on the ITestDelegateBridge interface?
I see it as just uneccessary boilderplate code

Reply:
I replaced `ITestDelegateBridge` with `IDelegateBridge`, where possible, in 845b9f977.  This should at least minimize its usage further until we can finally eliminate it.